pkg/gitcredentials/gitcredentials.go

```diff
@@ -257,3 +267,18 @@ func removeCredentialHelper(content string) string {
 
 	return strings.Join(out, "\n")
 }
+
+// getGlobalGitConfigPath resolves the global git config for the specified user according to
+// https://git-scm.com/docs/git-config/#Documentation/git-config.txt-XDGCONFIGHOMEgitconfig
+func getGlobalGitConfigPath(userName string) (string, error) {
+	if xdgConfigHome := os.Getenv("XDG_CONFIG_HOME"); xdgConfigHome != "" {
+		return filepath.Join(xdgConfigHome, ".gitconfig"), nil
```

Comment:
@pascalbreuninger Shouldn't this be, according to the documentation linked:

```go
return filepath.Join(xdgConfigHome, "git", "config"), nil
```

Also, both files `$XDG_CONFIG_HOME/git/config` and `~/.gitconfig` are read by git in order. Not sure if relevant here.

Reply:
Have opened a PR for this gh-1183